docs/configuration/firewall/index.rst

```diff
@@ -155,34 +147,12 @@ configuration:
    ipv4
    ipv6
    flowtables
-   zone
 
 .. note:: **For more information**
    of Netfilter hooks and Linux networking packet flows can be
    found in `Netfilter-Hooks
    <https://wiki.nftables.org/wiki-nftables/index.php/Netfilter_hooks>`_
 
-***************
-Legacy Firewall
-***************
-
-.. toctree::
-   :maxdepth: 1
-   :includehidden:
-
-   general-legacy
```

Comment:
would you also delete the general-legacy.rst and fix the refs to this file.

Reply:
File removed